images/manageiq-base-worker/Dockerfile

```diff
@@ -9,6 +9,17 @@ LABEL name="manageiq-base-worker" \
 
 COPY container-assets/entrypoint /usr/local/bin
 COPY container-assets/manageiq_liveness_check /usr/local/bin
+COPY container-assets/VMware-* /tmp/
+
+RUN cd /tmp && \
```

Comment:
Can you add a comment here, something like:

# Install the VDDK if found in `/tmp/VMware-*` ?  (Aside, we may want to provide an ARG, defaulted to `/tmp/VMware-*` instead of guessing the location)

cc @agrare

Reply:
Oh sorry...I misunderstood that the caller puts it in the container-assets, and it's copied _into_ /tmp inside the container.  This is fine.

Would be nice to avoid the layer that has the VDDK in it however I'm not sure how.

Reply:
Maybe use a multi-stage build and COPY it into the first stage?